prototypes/fulcro/src/avisi/apps/tech_testing_ground/prototypes/fulcro/client/current_app.cljs

```diff
@@ -1,33 +1,34 @@
 (ns avisi.apps.tech-testing-ground.prototypes.fulcro.client.current-app
   (:require
-    [com.fulcrologic.fulcro.application :as app]
-    [com.fulcrologic.fulcro.components :as comp]
-    [com.fulcrologic.fulcro.networking.http-remote :as http-remote]
-    [com.fulcrologic.fulcro.data-fetch :as df]
-    [avisi.apps.tech-testing-ground.prototypes.fulcro.client.hello-world :as root]))
-
-(defonce
-  app
-  (app/fulcro-app
-    {:remotes
-       {:remote
-          (http-remote/fulcro-http-remote
-            {:request-middleware
-               (->
-                 (http-remote/wrap-fulcro-request
-                   (fn [req] (assoc-in req [:headers "Accept"] "application/transit+json"))))})}}))
-(defn ^:export init
-  "Shadow-cljs sets this up to be our entry-point function. See shadow-cljs.edn `:init-fn` in the modules of the main build."
-  []
-  (app/mount! app root/Root "app")
-  (df/load! app :hello-world root/HelloWorld)
-  (js/console.log "Loaded"))
+    [com.fulcrologic.fulcro.application :as f-app]
+    [com.fulcrologic.fulcro.networking.http-remote :as http-remote]))
 
+(defonce ^:private current-app (atom nil))
+(defonce ^:private mount-element-id "app")
+(def ^:private remotes
+  {:remote
+     (http-remote/fulcro-http-remote
+       {:request-middleware
+          (->
+            (http-remote/wrap-fulcro-request
+              (fn [req] (assoc-in req [:headers "Accept"] "application/transit+json"))))})})
+(defn ^:private reset-current-app [{:keys [client-did-mount]}]
```

Comment:
For `defn` you can use `defn-` if you want to have it private. We also don't really use private that much to be honest. But I am not against it.